<launch>
    <node name="mpu6050_node" pkg="rpi_mpu6050" type="mpu6050_node" output="screen">
        <!--param name="sample_rate" type="int" value="10"/-->
        <!--param name="frame_id" type="str" value="imu_link" /-->
        <!--param name="ado" type="bool" value="false" /-->
    </node>
</launch>
